src/batch-transaction.ts

```diff
@@ -284,7 +292,17 @@ class BatchTransaction extends Snapshot {
    * @param {CreateReadPartitionsCallback} [callback] Callback function.
    * @returns {Promise<CreateReadPartitionsResponse>}
    */
-  createReadPartitions(options, callback) {
+  createReadPartitions(
+    options: ReadRequest,
+  ): Promise<CreateReadPartitionsResponse>;
+  createReadPartitions(
+    options: ReadRequest,
+    callback: CreateReadPartitionsCallback,
+  ): void;
+  createReadPartitions(
+    options: ReadRequest,
+    cb?: CreateReadPartitionsCallback,
```

Comment:
why the parameter name is different here. In above signature its callback , here its cb. How do we know in method which variable has to be used ?
Same is applicable for createQueryPartitions

Reply:
Understood, thanks for clarification